docs/monitoring/shell-to-node.md

```diff
@@ -0,0 +1,424 @@
+---
+title: Get a Shell to a Gardener Shoot Worker Node
+description: "Describes the methods for getting shell access to worker nodes."
```

Comment:
This should not be in the docs/monitoring, instead I think it should be in https://github.com/gardener/gardener/tree/master/docs/usage.

And having in mind https://github.com/gardener/gardener/blob/master/docs/proposals/15-manage-bastions-and-ssh-key-pair-rotation.md, I think it will need to be re-worked. 

Also, it is heavily using gardenctl, so I would assume it is better in the gardenctl repo.